src/dbnode/encoding/proto/vehicle_location_2.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+message VehicleLocation {
```

Comment:
can you add tests:
1. no schema change, but schema has with proto values (non-custom fields)
2. no schema change, but there is time unit change
3. no time unit change, has schema change, custom fields change, but proto values do not change
4. no time unit change, schema change, custom fields do not change, but proto values do change
5. no time unit change, schema change, custom fields change, proto values change. 

for case 3, should we expect the control bit per write to be 1XXX?

Reply:
Sure. The existing tests cover #1 (TestRoundTripPropTest generates schemas that have non-custom fields),  #2 (TestRoundTripProp generates a unique time unit for each write) and #3 (TestRoundTripMidStreamSchemaChanges).

#4 and #5 you're right I should add some tests.

Reply:
Sync'd in person, will add a unit test that includes non-custom fields too for easier debugging